resources/views/volumes/clone.blade.php

```diff
@@ -0,0 +1,195 @@
+@extends('app')
+
+@section('title', 'clone volume')
+
+@push('scripts')
+    <script type="text/javascript">
+        biigle.$declare('destinationProjects', {!!$destinationProjects!!});
+        biigle.$declare('volume', {!!$volume!!});
+        biigle.$declare('name', '{!!old('name',$volume->name)!!}');
+        biigle.$declare('fileLabelTrees', '{!!$labelTrees!!}');
+        biigle.$declare('selectedFilesIds', '{!! collect(old('only_files',[])) !!}');
+        biigle.$declare('selectedFiles', '{!! collect(old('selected_files',[])) !!}');
+        biigle.$declare('fileLabelIds', '{!! collect(old('only_file_labels',[])) !!}');
+        biigle.$declare('annotationLabelTrees', '{!!$labelTrees!!}');
+        biigle.$declare('cloneFileLabels', {{old('clone_file_labels',false)}});
+        biigle.$declare('cloneAnnotations', {{old('clone_annotations',false)}});
+        biigle.$declare('annotationLabelIds', '{!! collect(old('only_annotation_labels', [])) !!}');
```

Comment:
In our last meeting we agreed that adding the label trees as JSON strings and using `JSON.parse()` would be useful because this way we get two independent label trees objects and the label `selected` state would not be shared across them. But this is wrong (I see now), you don't need strings and `JSON.parse()` at all for this. It's enough to define both objects as separate varaibles here. Also, you don't need strings and `JSON.parse()` for any of the other variables:

```suggestion
        biigle.$declare('fileLabelTrees', {!!$labelTrees!!});
        biigle.$declare('selectedFilesIds', {!! collect(old('only_files',[])) !!});
        biigle.$declare('selectedFiles', {!! collect(old('selected_files',[])) !!});
        biigle.$declare('fileLabelIds', {!! collect(old('only_file_labels',[])) !!});
        biigle.$declare('annotationLabelTrees', {!!$labelTrees!!});
        biigle.$declare('cloneFileLabels', {{old('clone_file_labels',false)}});
        biigle.$declare('cloneAnnotations', {{old('clone_annotations',false)}});
        biigle.$declare('annotationLabelIds', {!! collect(old('only_annotation_labels', [])) !!});
```